### PR TITLE
Voice channel mention

### PR DESCRIPTION
**Pull request by yo-louk**
### This pull request...
  - [ ] Fixes a bug
  - [ ] Introduces a new feature
  - [x] Improves an existing feature
  - [ ] Boosts code quality or performance

### Description
I recently noticed voice channels can be mentioned just like text channels. A small change to the code can implement this feature.

image
The top example is the current way the bot would mention a channel, the bottom is what I wish it would look like.
This is my first pull request. I have not tested this change as I consider it to be too simple and don't expect it to fail.

### Purpose
Aesthetics and ease of use: clicking on the mentioned channel will automatically connect you to it.

### CODE REVIEW

File: src/main/java/com/jagrosh/jmusicbot/commands/MusicCommand.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ protected void execute(CommandEvent event)
             GuildVoiceState userState = event.getMember().getVoiceState();
             if(!userState.inVoiceChannel() || userState.isDeafened() || (current!=null && !userState.getChannel().equals(current)))
             {
-                event.replyError("You must be listening in "+(current==null ? "a voice channel" : "**"+current.getName()+"**")+" to use that!");
+                event.replyError("You must be listening in "+(current==null ? "a voice channel" : "<#"+current.getId()+">")+" to use that!");
```
Comment (MichailiK) [inline src/main/java/com/jagrosh/jmusicbot/commands/MusicCommand.java:73]:
```suggestion
                event.replyError("You must be listening in "+(current==null ? "a voice channel" : current.getAsMention())+" to use that!");
```

File: src/main/java/com/jagrosh/jmusicbot/commands/MusicCommand.java [hunk 2]
```diff
@@ -89,7 +89,7 @@ protected void execute(CommandEvent event)
                 }
                 catch(PermissionException ex) 
                 {
-                    event.reply(event.getClient().getError()+" I am unable to connect to **"+userState.getChannel().getName()+"**!");
+                    event.reply(event.getClient().getError()+" I am unable to connect to <#"+userState.getChannel().getId()+">!");
```
Comment (MichailiK) [inline src/main/java/com/jagrosh/jmusicbot/commands/MusicCommand.java:92]:
```suggestion
                    event.reply(event.getClient().getError()+" I am unable to connect to "+userState.getChannel().getAsMention()+"!");
```

File: src/main/java/com/jagrosh/jmusicbot/commands/admin/SetvcCmd.java [hunk 3]
```diff
@@ -62,7 +62,7 @@ else if (list.size()>1)
             else
             {
                 s.setVoiceChannel(list.get(0));
-                event.reply(event.getClient().getSuccess()+" Music can now only be played in **"+list.get(0).getName()+"**");
+                event.reply(event.getClient().getSuccess()+" Music can now only be played in <@"+list.get(0).getId()+">");
```
Comment (MichailiK) [inline src/main/java/com/jagrosh/jmusicbot/commands/admin/SetvcCmd.java:65]:
```suggestion
                event.reply(event.getClient().getSuccess()+" Music can now only be played in "+list.get(0).getAsMention());
```

**Review by MichailiK — CHANGES_REQUESTED** [review]
JDA offers a method to get a channel's mention string. Use that method instead.

**Comment by MichailiK** [comment]
Additionally, my IDE has found another 3 usages that you've missed:

https://github.com/jagrosh/MusicBot/blob/b83028fb97318825c7adaa12caa4a3b94513c652/src/main/java/com/jagrosh/jmusicbot/audio/AudioHandler.java#L205

https://github.com/jagrosh/MusicBot/blob/b83028fb97318825c7adaa12caa4a3b94513c652/src/main/java/com/jagrosh/jmusicbot/commands/general/SettingsCmd.java#L60

https://github.com/jagrosh/MusicBot/blob/b83028fb97318825c7adaa12caa4a3b94513c652/src/main/java/com/jagrosh/jmusicbot/utils/FormatUtil.java#L77

Update these accordingly.

**Comment by yo-louk** [comment]
Thanks for the elaborate feedback.
I agree with `getAsMention()` being more appropriate than `"<#"+vc.getId()+">"`.
I also applied these changes to the 3 other usages that I had missed earlier.

After successfully compiling the project, I ran the commands and got some good results, a few of which you can see below:

image

**Review by MichailiK — APPROVED** [review]
Looks good! @jagrosh will have to take another look & merge if everything looks good.

**Pull request merged**